src/amp/components/Epic.tsx

```diff
@@ -9,6 +9,7 @@ import {
 } from '@root/src/amp/components/moustache';
 import { palette } from '@guardian/src-foundations';
 import { headline, body, textSans } from '@guardian/src-foundations/typography';
+import {Ticker} from "@root/src/amp/components/Ticker";
```

Comment:
I think that prettier doesn't seem to be working, it might be worth configuring it to run on save.

Reply:
✅